tripper/triplestore.py

```diff
@@ -646,3 +658,55 @@ def _add_function_fno(self, func, expects, returns, base_iri):
             lst = lst_next
 
         return func_iri
+
+    def _add_function_emmo(self, func, expects, returns, base_iri):
+        """Implementing add_function() for EMMO."""
```

Comment:
Do we need a docstring?

Reply:
This is an EMMO-specific implementation of the public method `add_function()`, which is documented above. 

I will add a few more words...